### ISSUE TITLE
DCA enchancements

### ISSUE THREAD

**Issue by RunPlz** [taniman/profit-trailer-enhancements#92]
Situation: PAIR gets too many DCA
Would be nice to be able to split it into smaller bags so that it may get tackled down with less BTC effort reducing the required BTC designated to a specific bag.

**Comment by Seeletse** [comment]
Telegram notification for DCA re-buy would be nice.

**Comment by sas7085** [comment]
In DCA i should be able to decide the % of Buy  not like today it is 100% or your available max fund in case of partial fill. Not always Double favors everyone. In case of less budget and not sure which coin will bounce back quickly or no hope of that coin then it will be better my DC should re buy <some % which i can set in config file>  .   For example  i need to watch before consuming all my budget in 1 or 2 coins in DCA i will choose go for  25 %  rebuy in DCA  not double the coins  . If market will go too much down i will try to save my BTC . when the market  bounce back i will go for more % DCA .

**Comment by sas7085** [comment]
Sometimes we are getting this Error " There are not enough coins in the orderbook to do a FULL DCA " . The system also not doing partial DCA .  When  we should have option to choose  partial  or  full DCA.

**Comment by syl201971** [comment]
Or a manual buy option

**Comment by dutchy1966** [comment]
I would like an option to "tag" a coin pair for DCA. DCA can easily eat your balance and get you trapped with all your money stuck in DCA and no more possibilty to buy new pairs. If there is a tag option you could select one or a couple of coins to DCA. After they are cleared some other coins can be selected for DCA. That way you keep way more balance for buying and selling. Clear bags a few at a time instead of all in parallel.

**Comment by clownfish44** [comment]
You can already do this by just moving part of the coin off of the exchange which is the best way because if you bought more and had all of it on the exchange you would be averaging all of it on the exchange which would not help.

**Comment by RunPlz** [comment]
Alright, so I if I now have 90% funds on one DCA pair your suggestion is to remove coin balance and pay for transference fees every single time until I get to deal with this rather than have the bot "understand" I would like to "work" with a lower amount coin balance.

Would this be achievable by placing a stop order manually?

Is there any other work-around you may suggest?
